components/x-teaser/storybook/top-story.js

```diff
@@ -17,6 +17,7 @@ exports.knobs = [
 	'metaPrefixText',
 	'metaSuffixText',
 	'metaLink',
+	'dataTrackable',
```

Comment:
This only shows for the top story teaser in Storybook and not the other teasers, I'm not sure how important it is for it to show up for all of them. Was there a specific reason you only added to `top-story` @bjfletcher?

Reply:
No good reason (we're currently analysing only top stories) and I think that's a good idea to add to all the other types for the storybook. So I've just pushed a commit to add the knob to all the other teaser types.